src/utils.js

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
i'm a bit of a stickler, but `utils` has always bothered me. :) Can we rename this to `assignment-ast.js` to denote that its purpose is for finding assignment values. :)

Reply:
Makes sense to me, renamed here https://github.com/testing-library/eslint-plugin-jest-dom/pull/122/commits/94aad94414dd435f14c4c40153dbcf39726e1b60